Gemfile

```diff
@@ -3,6 +3,8 @@ source 'https://rubygems.org'
 # Please see sufia.gemspec for dependency information.
 gemspec
 
+gem 'curation_concerns', github: 'projecthydra/curation_concerns', branch: 'master'
```

Comment:
Just as a note to @awead, I don't think there's any need for this if you put it in Gemfile.extra.